pattern/index.js

```diff
@@ -20,6 +24,25 @@ function generatorPattern() {
     }
   }
 
+  function promptForName() {
```

Comment:
Added the `promptForName` so that if the user forgets to provide a name, they are prompted instead of an error occurring.